### DISCUSSION TITLE
1.18?

### DISCUSSION THREAD

**Discussion by Patssssss** [Jikoo/OpenInv#54] [General]
Will this be available in 1.18 soon?

**Comment by Jikoo** [comment]
Dev builds are already available. Please see #52

**Reply by Patssssss** [reply]
Any idea how i can use that on aternos server? im sorry im new to this kind of things.

**Reply by Jikoo** [reply]
Nope. Review your host's documentation for how to upload plugins, and if you're still having trouble, contact their support.

**Reply by Patssssss** [reply]
Thnx!
